docs/3.0rc/resources/daemonize-processes.mdx

```diff
@@ -133,7 +133,7 @@ def say_hi():
     print("Hello!")
 
 if __name__=="__main__":
-    say_hi.serve(name="Greeting from daemonized .serve")
```

Comment:
i dont think we should encourage weird deployment names (like containing periods etc)